release/env/env.go

```diff
@@ -21,14 +21,20 @@ func mustGet(varname string) (string, error) {
 	return val, nil
 }
 
+// EvgIsPatch returns whether the current evergreen task is a patch,
+// based on the value of an env variable set by the evg project file.
 func EvgIsPatch() bool {
 	return get("EVG_IS_PATCH") != ""
 }
 
+// EvgIsPatch returns the build_id of the current evergreen task,
```

Comment:
These all say `EvgIsPatch` :)

Reply:
fixed! That's what I get for trying to rush out the update before standup :)